website/docs/maintenance/backup_restore.md

```diff
@@ -10,7 +10,7 @@ If you are using `docker-compose`, you are using Docker Compose v1, which has be
 Create backup file `teslamate.bck`:
 
 ```bash
-docker compose exec -T database pg_dumpall -U teslamate > /backuplocation/teslamate.bck
+docker compose exec -T database pg_dump -U teslamate teslamate > /backuplocation/teslamate.bck
```

Comment:
the `-U teslamate teslamate` has a duplicate

Reply:
Hmmm, that is the database name:
```
dbname
Specifies the name of the database to be dumped. If this is not specified, the environment variable PGDATABASE is used. If that is not set, the user name specified for the connection is used.
```
So, yes, if the user name is same as database name, it could be left out.

Reply:
You are right, in the first commits it was removed which confused me